src/java/picard/sam/CompareSAMs.java

```diff
@@ -50,11 +50,23 @@
  * @author alecw@broadinstitute.org
  */
 @CommandLineProgramProperties(
-        usage = "USAGE: CompareSAMS <SAMFile1> <SAMFile2>\n" +
-                "Compares the headers of the two input SAM or BAM files, and, if possible, the SAMRecords. " +
+        usage = "USAGE: CompareSAMS <SAMFile1> <SAMFile2>" +
+                "Compares the headers of the two input SAM or BAM files and if possible, the SAMRecords. " +
                 "For SAMRecords, compares only the readUnmapped flag, reference name, start position and strand. " +
```

Comment:
I would start by covering the header comparison, then the samrecord comparison. More intuitively logical that way.